packages/bot-web-ui/src/stores/app-store.js

```diff
@@ -25,13 +25,100 @@ export default class AppStore {
         this.api_helpers_store = null;
     }
 
+    getErrorForNonEuClients = () => ({
+        text: localize(
+            'Unfortunately, this trading platform is not available for EU Deriv account. Please switch to a non-EU account to continue trading.'
+        ),
+        title: localize('Deriv Bot is unavailable for this account'),
+        link: localize('Switch to another account'),
+    });
+
+    getErrorForEuClients = (is_logged_in = false) => {
+        return {
+            text: ' ',
+            title: is_logged_in
+                ? localize('Deriv Bot is not available for EU clients')
+                : localize('Deriv Bot is unavailable in the EU'),
+            link: is_logged_in && localize("Back to Trader's Hub"),
+            route: routes.traders_hub,
+        };
+    };
+
+    handleErrorForEu = (show_default_error = false) => {
+        const { client, common, ui, traders_hub } = this.core;
+        const toggleAccountsDialog = ui?.toggleAccountsDialog;
+        const recovery_low_risk_cr_eu = traders_hub?.selected_region;
```

Comment:
@maryia-matskevich-deriv  isn't it better to rename this variable to some thing more descriptive?

Reply:
`recovery_low_risk_cr_eu` was removed, the reason you can find in the next thread